Misc/ACKS

```diff
@@ -1788,3 +1788,4 @@ Jelle Zijlstra
 Gennadiy Zlobin
 Doug Zongker
 Peter Åstrand
+Shang Hui
```

Comment:
Could you move this line with other S or H names?  (It’s loosely sorted by last name, for names that have a concept of last name)

Reply:
OK, I will do it.